recipes/r-bpcells/meta.yaml

```diff
@@ -0,0 +1,164 @@
+{% set version = '0.3.0' %}
+{% set github = 'https://github.com/bnprks/BPCells' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-bpcells
+  version: {{ version|replace('-', '_') }}
+
+source:
+  url: {{ github }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 231e91a3d60d2bc400324c5b60fb4644fda16261048d6b1a49bfac4aa269cd1f
+  patches:
+    # track: https://github.com/bnprks/BPCells/issues/228
+    - patches/0001-fix-libhwy-config.patch
+    # from: https://github.com/bnprks/BPCells/pull/231
+    - patches/0002-granges-conversion.patch
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  run_exports:
+    - {{ pin_subpackage('r-bpcells', max_pin='x.x') }}
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - cross-r-base {{ r_base }}  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - r-base
+    - r-dplyr >=1.0.0
+    - r-ggplot2 >=3.4.0
+    - r-ggrepel
+    - r-hexbin
+    - r-lifecycle
+    - r-magrittr
+    - r-matrix
+    - r-patchwork
+    - r-rcolorbrewer
+    - r-rcpp
+    - r-rcppeigen
+    - r-readr
+    - r-rlang
+    - r-scales
+    - r-scattermore
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-vctrs
+    - hdf5
+    - libhwy
+    - zlib
+  run:
+    - r-base
+    - r-dplyr >=1.0.0
+    - r-ggplot2 >=3.4.0
+    - r-ggrepel
+    - r-hexbin
+    - r-lifecycle
+    - r-magrittr
+    - r-matrix
+    - r-patchwork
+    - r-rcolorbrewer
+    - r-rcpp
+    - r-readr
+    - r-rlang
+    - r-scales
+    - r-scattermore
+    - r-stringr
+    - r-tibble
+    - r-tidyr
+    - r-vctrs
+
+test:
+  requires:
+    - r-testthat
+    - bioconductor-genomicranges
+    - bioconductor-iranges
+    - r-igraph
+    - r-matrixstats
+    - r-rspectra
+    - r-withr
+    - macs3
+  commands:
+    - $R -e "library('BPCells')"
+    ## NB: this tightly-couples to `macs2` coming from `test.requires` and avoids running against the container
+    - if command -v macs3; then $R -e "testthat::test_package('BPCells', stop_on_failure=TRUE)"; fi
+
+about:
+  home: https://bnprks.github.io/BPCells
+  dev_url: {{ github }}
+  license: Apache-2.0 or MIT
```

Comment:
I've never seen this dual-license combo before, but it is clearly what upstream intends, so I agree with using `or` and bundling all the license files into the conda binary (comment only ie no action required)